CMakeLists.txt

```diff
@@ -251,6 +251,13 @@ else()
 	endif( APPLE )
 endif()
 
+
+	if(HAIKU)
+		# Haiku networking resides in libnetwork
+		set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lnetwork")
+		set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -lnetwork")
```

Comment:
Does this really needed? We have no shared libraries. And what’s the point of the indent?

Reply:
Yes, sorry, the CMAKE_SHARED_LINKER_FLAGS can probably be omitted. I just copied it from some other patch.